src/main/java/com/easypost/model/ErrorDeserializer.java

```diff
@@ -36,18 +60,18 @@ public Error deserialize(final JsonElement json, final Type typeOfT,
             error.setCode("NO RESPONSE CODE");
             return error;
         }
-        
-        JsonElement errorMessage = results.getAsJsonObject().get("message");
-        if (errorMessage.isJsonArray()) {
-            JsonArray jsonArray = errorMessage.getAsJsonArray();
-            ArrayList<String> messages = new ArrayList<>();
-
-            for (int i = 0; i < jsonArray.size(); i++) {
-                messages.add(jsonArray.get(i).getAsString());
-            }
 
+        try {
+            ArrayList<String> messages = new ArrayList<>();
+            JsonElement errorMessageJson = results.getAsJsonObject().get("message");
```

Comment:
question: how does this work if it's a list? We are trying to get it as an object so will that mean we no longer capture when something is a list?

Reply:
It will take the value of the "message" key and turn it into a `JsonObject`, the root of any JSON element for the GSON library. In the `traverseJsonElement` function, it will handle if it's a list/array.

Reply:
Nate is correct, everything is being handled in the `traverseJsonElement` function recursively.